README.md

```diff
@@ -503,7 +536,66 @@ sudo docker run --rm \
     simplelogin/app:3.2.2 python init_app.py
 ```
 
-Now, it's time to run the `webapp` container!
+now it time to setup the `webapp` and `email handler` containers
+
+for docker-compose add this to your file:
+
+```
+    simplelogin:
+        image: simplelogin/app:3.2.2
+        container_name: sl-app
+        ports:
+            - "7777:7777"
+        restart: unless-stopped
+        volumes:
+        # this is to add more words to create more unique aliases
+        # - ./words.txt:/code/local_data/words.txt
```

Comment:
by default, words are retrieved from `local_data/words_alpha.txt` which contains more words than the `local_data/words.txt` which is rather used for local development.

Reply:
interesting, it worked for me and the aliases used the new custom words
in the example.env is says it uses words.txt:
https://github.com/simple-login/app/blob/2d9abe55a448a5ece9c4b84e2eebf365c168f18e/example.env#L110

doesn't the example file use the default value?

Reply:
fixed https://github.com/simple-login/app/pull/309/commits/b374f5620091577d57b5d04322aaaf76aa291ad6